pages/components/SIG/DesktopSIG.tsx

```diff
@@ -0,0 +1,110 @@
+										<p className='text-xxs px-12 text-justify text-white'>
+											Programming adalah sebuah proses untuk membuat suatu
+											program di komputer yang dapat berupa software, website,
+											aplikasi, dan lain sebagainya. Proses ini melibatkan
+											beberapa tahap pemrograman, mulai dari tulis menulis,
+											menguji, merevisi, dan mengevaluasi, serta mengujinya lagi
+											sampai program tersebut benar-benar sesuai dengan apa yang
+											diinginkan oleh seorang programmer. Pada perlombaan
+											Programming, biasanya kita akan dihadapi dengan
+											permasalahan IT yang akan menguji kemampuan dan nalar kita
+											untuk dapat menyelesaikannya dengan suatu program
+											komputer.{' '}
+										</p>
+									</div>
+									<div className='container w-full h-1/4 flex justify-center items-center'>
+										<div className='container h-1/2 w-48 mt-10 lg:h-full lg:w-full lg:mt-0 flex flex-col p-5 justify-end items-center'>
+											<button className=' object-center flex m-15 flex-no-shrink fill-current'>
+												<Image src={JoinSIG} />
+											</button>
+										</div>
+									</div>
+								</div>
+							</div>
+							<div className='mx-5 container h-full w-full  relative '>
+								<Image src={DesktopTicketBackground} className='' />
+								<div className='container w-full h-full  z-10 absolute top-0'>
+									<div className='container w-full h-1/4  text-center flex flex-col justify-center items-center content-center'>
+										<p className='text-gray-400'>IT Interest</p>
+										<h1 className='text-white text-2xl'>Programing</h1>
+									</div>
+									<div className='container w-full h-2/4  overflow-hidden '>
+										<p className='text-xxs px-12 text-justify text-white'>
+											Programming adalah sebuah proses untuk membuat suatu
+											program di komputer yang dapat berupa software, website,
+											aplikasi, dan lain sebagainya. Proses ini melibatkan
+											beberapa tahap pemrograman, mulai dari tulis menulis,
+											menguji, merevisi, dan mengevaluasi, serta mengujinya lagi
+											sampai program tersebut benar-benar sesuai dengan apa yang
+											diinginkan oleh seorang programmer. Pada perlombaan
+											Programming, biasanya kita akan dihadapi dengan
+											permasalahan IT yang akan menguji kemampuan dan nalar kita
+											untuk dapat menyelesaikannya dengan suatu program
+											komputer.{' '}
+										</p>
+									</div>
+									<div className='container w-full h-1/4 flex justify-center items-center'>
+										<div className='container h-1/2 w-48 mt-10 lg:h-full lg:w-full lg:mt-0 flex flex-col p-5 justify-end items-center '>
+											<button className=' object-center flex m-15 flex-no-shrink fill-current '>
+												<Image src={JoinSIG} />
+											</button>
+										</div>
+									</div>
+								</div>
+							</div>
+						</div>
```

Comment:
this could be refactored into separated components and you could do a mapping instead of write it repeatedly